apps/public/src/app/(map)/locatie/kies/_components/SideBar/SideBar.tsx

```diff
@@ -40,10 +94,33 @@ export const SideBar = ({ coordinates }: Props) => {
         </Paragraph>
       </div>
       <div>
-        <Heading level={2} size="level-4">
-          Zoek op adres
-        </Heading>
-        {address && <div>{address}</div>}
+        <Field>
+          <HUILabel as={Label}>Zoek op adres</HUILabel>
+          <Combobox value={address} onChange={setAddress} onClose={() => fetchAddressList('')}>
+            <ComboboxInput
+              as={TextInput}
+              displayValue={(item: Address) => item?.weergave_naam}
+              name="address"
+              onChange={(event) => fetchAddressList(event.target.value)}
+              autoComplete="off"
+            />
+            {showListBox && (
+              <ComboboxOptions as={ListBox} modal={false}>
```

Comment:
@vdegraaf wrote: On mobile it pushes the map to the bottom when the options appear. Resulting in very strange UX.

You're right, I didn't want to touch the SideBar styling too much in this PR, seeing as that will probably conflict with [this ticket](https://gemeente-amsterdam.atlassian.net/jira/software/c/projects/SIG/boards/1243?selectedIssue=SIG-6268). But the SideBar should probably scroll when taller than the viewport. Do you want to handle that now?

Reply:
Hmm, I don't think it should touch upon the map or list right? The idea is that it flows over this right?

Reply:
The ListBox is supposed to push the following content down (see also the gov uk example). When it does that near the bottom edge, it makes the SideBar taller than `100vh`. SideBar should probably be fixed to `100vh` and scroll if its content gets taller.